Description: Drop ryu from dependencies
 It supports an alternative ovs driver implementation; its
 not the default, so deferring hard dependency for now
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

--- a/requirements.txt
+++ b/requirements.txt
@@ -19,7 +19,6 @@
 neutron-lib>=0.0.1 # Apache-2.0
 python-neutronclient>=2.6.0 # Apache-2.0
 retrying!=1.3.0,>=1.2.3 # Apache-2.0
-ryu>=3.30 # Apache-2.0
 SQLAlchemy<1.1.0,>=1.0.10 # MIT
 WebOb>=1.2.3 # MIT
 keystoneauth1>=2.1.0 # Apache-2.0
